apps/classifications/api.py

```diff
@@ -0,0 +1,65 @@
+from django.shortcuts import get_object_or_404
+from rest_framework import mixins
+from rest_framework import viewsets
+
+from adhocracy4.api.permissions import ViewSetRulesPermission
+from adhocracy4.projects.models import Project
+from apps.classifications.models import AIClassification
+from apps.classifications.models import UserClassification
+from apps.projects import helpers
+
+from . import serializers
+
+
+class UserClassificationViewSet(mixins.ListModelMixin,
```

Comment:
As only the upper part is different for both ClassificationViewSets, you could make one, both inherit from. Or maybe that's only, when you use the code the third time. :)

Reply:
Sure, can do that